tests/unit/llvm_lit_tests/InlineDivergentScopes.ll

```diff
@@ -40,6 +40,8 @@ block_e:
 ; CHECK: block_c_ids:
 ; CHECK-NEXT:   br label %block_b
 
+; IDS on switch test
+
```

Comment:
Please squash this commit with the one implementing the new IDS on multiple divergent scopes.